README.md

```diff
@@ -43,8 +43,8 @@ The minimal recommended configuration file is:
 listen = "<ip-address>:<port>";
 
 github {
-  reposdir = "<path>";
   match ".*" {
+    cmd = "/path/to/prps/%o/%r %e %j";
```

Comment:
What is `prps` here? Per-repo programs? I think we ditched that terminology a while ago didn't we?

Reply:
We didn't ditch the terminology, but I'm open to suggestions of alternative names, if it makes things clearer. Note that the README does now say something along the lines of "one way of organising things is as per-repo programs."

Reply:
I think i might be wrong here, as i've since seen lots of references to that term.

I thought `reposdir` had replaced `per-repo-programs`. Are we mixing terminology to freely perhaps?

Reply:
Originally, a `reposdir` would contain "per repo programs", and that structure was enforced by snare-0.1.0. This PR removes the enforcement of either concepts, but that doesn't mean that the terminology isn't vaguely useful, since I think most people (or, at least us), will structure things as before.

Reply:
(sorry, we raced, so my last comment seems out of context).

So are is the `per-repo-programs` directory different from the `reposdir`? I think so. In which case we need separate terms anyway.

Reply:
Yes.